pvlib/atmosphere.py

```diff
@@ -337,6 +337,82 @@ def gueymard94_pw(temp_air, relative_humidity):
     return pw
 
 
+def rh_from_tdew(temperature, dewpoint, coeff=(6.112, 17.62, 243.12)):
+    """
+    Calculate relative humidity from dewpoint temperature using the Magnus equation.
```

Comment:
```suggestion
    Calculate relative humidity from dew-point temperature using the Magnus equation.
```

Reply:
(inner grammarian emerges) the term is either "dewpoint" or "dew point": [NOAA glossary](https://forecast.weather.gov/glossary.php?word=DEW#:~:text=It%20is%20the%20temperature%20to,as%20one%20word%20(Dewpoint).). The AMS glossary prefers "dewpoint".

I can't find any instances of "dew-point" as a hyphenated adjective, although I understand that's what grammar rules suggest it should be.

Reply:
Admittedly I didn't search beyond the WMO document we referenced, so it could be an old-world / new-world thing.

Reply:
Happy to use any term that you all prefer

Reply:
Looks like EPW and TMY3 files use the hyphen.  Most of the pvlib docs use "dew point", with the exception of the Nomenclature page which says "Dewpoint".  

I suggest we leave this for a potential package-wide cleanup and not worry about it here.